wiki/Best_Of/2016/en.md

```diff
@@ -0,0 +1,1865 @@
+# 2016
+
+**_[Click here to return to the Best of main page](../ "Best Of main page")_**
+
+## General
+
+- [News Results](https://osu.ppy.sh/news/157306073998)
+
+### Weighting formula
+
+This year’s weighted score was calculated taking into account all the scores the voter has on all the beatmaps within the given set. For each score, we use the following formula to calculate a weighting:
+
+![Formula of calculating a weighting](2016formula.png "Formula of calculating a weighting")
```

Comment:
I don't know about the new article but I believe this should be `Formula for calculating weighting`